### PR TITLE
Css changes to the user search page

### PR DESCRIPTION
**Pull request by roberthouse54**
#### What are the relevant tickets?
#934

#### What's this PR do?
This fixes just some of the css styles for this issue. There are other changes that need to be made that go beyond just changing the styles.

#### Where should the reviewer start?
Verify that my css changes are made on the correct classes, etc.

#### How should this be manually tested?
See that the table of results, and the filters are not broken.

### CODE REVIEW

**Comment by aliceriot** [comment]
nothing looks broken or off! looking at the code now.

File: static/scss/_button.scss [hunk 1]
```diff
@@ -22,7 +22,7 @@
   &.minor-action {
     font-size: 13px;
     padding: 9px 19px;
-    border: 1px solid #e6e6e6;
+    border: 1px solid #d0d0d0;
```
Comment (aliceriot) [inline static/scss/_button.scss]: do you think we'll want to use this color elsewhere?

File: static/scss/search_page.scss [hunk 2]
```diff
@@ -1,6 +1,15 @@
 @import "../../node_modules/searchkit/theming/components";
 
 .learners-search {
+  width: 97%;
+  margin: 0 auto;
+  max-width: 1400px;
+
+  .mdl-grid {padding: 0;
```
Comment (aliceriot) [inline static/scss/search_page.scss]: little formatting error, style rules should always be on their own line, like this:

```scss
.mdl-grid {
  padding: 0;
}
```

**Comment by aliceriot** [comment]
This generally looks good, but there are some formatting issues with the scss changes.

We generally try to leave a blank line after closing a block and before opening a new one. So instead of

```scss
.class {
  display: none;
  .other-class {
    color: purple;
  }
  .third-class {
    text-decoration: underline;
  }
}
```

we generally prefer:

```scss
.class {
  display: none;
  
  .other-class {
    color: purple;
  }
  
  .third-class {
    text-decoration: underline;
  }
}
```

when closing multiple blocks (like in the example when we close the `.third-class` and `.class` blocks) the curly braces don't need a blank line in between, and similarly when opening a block immediately inside another block a blank line is unnecessary, such as in this scenario:

```scss
.class {
  .inner-class {
    display: none;
    color: potato;
  }
}
```

lmk if that isn't clear! I figured it might be easier to give you a summary of the style guide than to go through and point out everything that should be changed.

File: static/scss/search_page.scss (hunk 2 above)
Comment (roberthouse54) [inline static/scss/search_page.scss]: OK, I fixed that

File: static/scss/_button.scss (hunk 1 above)
Comment (roberthouse54) [inline static/scss/_button.scss]: I changed this to $border-color, and added this variable to the minor-button and the pagination and sort controls.

**Comment by roberthouse54** [comment]
I think I fixed all the stuff.

File: static/scss/search_page.scss [hunk 3]
```diff
@@ -65,6 +80,7 @@
     }
 
     .filter-visibility-toggle.closed {
+      margin-bottom: 0px;
       i.closed {
```
Comment (aliceriot) [inline static/scss/search_page.scss]: can you add a blank line above this?

File: static/scss/search_page.scss [hunk 4]
```diff
@@ -23,36 +32,42 @@
...
+        margin: 15px 5px;
+        font-size: 15px;
       }
 
       .sk-hierarchical-menu-list__root {
-        margin-left: 25px;
-        margin-right: 40px;
+        margin-left: 6px;
+        margin-right: 0;
         .sk-hierarchical-menu-option {
```
Comment (aliceriot) [inline static/scss/search_page.scss]: ditto here

File: static/scss/search_page.scss [hunk 5]
```diff
@@ -23,36 +32,42 @@
...
-          margin-left: 25px;
-          margin-right: 40px;
+          align-items: stretch;
+          margin-left: 10px;
+          margin-right: 10px;
+        }
+        .sk-item-list-option__text {
+          margin-left: 10px;
         }
+        .sk-item-list-option__count {
```
Comment (aliceriot) [inline static/scss/search_page.scss]: here and on line 53 too

File: static/scss/search_page.scss [hunk 6]
```diff
@@ -23,36 +32,42 @@
...
     display: flex;
     flex-direction: column;
-    background-color: $fg-grey;
-
+    background-color: $bg-light-gray;
+    
+    .mdl-card {
+      padding: 10px;
+      background-color: $bg-light-gray;
+    }
```
Comment (aliceriot) [inline static/scss/search_page.scss]: blank line here

**Comment by aliceriot** [comment]
nvm disregard those comments, it looks like you didn't push your updated code?

**Comment by roberthouse54** [comment]
Damn, you are right. I'll do this in the morning. Sorry

**Comment by aliceriot** [comment]
LGTM 👍

**Pull request merged**